src/pages/Improbability/Improbability.utils.tsx

```diff
@@ -0,0 +1,38 @@
+import Box from '@mui/material/Box';
```

Comment:
utils reminds me of more of some small helper functions and stuff like that. Maybe lets change the file name to `.columns`?